libres/lib/include/ert/analysis/ies/ies_enkf.hpp

```diff
@@ -23,13 +23,30 @@
 #include <ert/util/rng.hpp>
 
 #include <ert/res_util/matrix.hpp>
+#include <ert/analysis/ies/ies_enkf_data.hpp>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void ies_enkf_linalg_store_active_W(ies_enkf_data_type *data,
+                                    const matrix_type *W0);
+
+matrix_type *ies_enkf_alloc_activeE(const ies_enkf_data_type *data);
+matrix_type *ies_enkf_alloc_activeW(const ies_enkf_data_type *data);
```

Comment:
Generally I would prefer if the `.hpp` included only only the external interface of `ies_enkf` to the remaining program. For testing purposes the prototype can be repeated in the corresponding cpp file. However, this is a design choice we have not discussed yet. Should we add a discussion for this @dotfloat ? Or do you have a better suggestion?